nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/CandidateBO.java

```diff
@@ -127,6 +127,10 @@ public URI publicationId() {
         return candidateDao.candidate().publicationId();
     }
 
+    PeriodStatus periodStatus() {
```

Comment:
this is needed to test status in avi-commons, unless we refactor public CandidateDto toDto()